Tests/SwiftIDEUtilsTest/Assertions.swift

```diff
@@ -0,0 +1,103 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import SwiftIDEUtils
+import SwiftParser
+import SwiftSyntax
+import _SwiftSyntaxTestSupport
+
+func assertClassification(
+  _ source: String,
+  expectedClassificationSpec: [ClassificationSpec] = []
```

Comment:
I don’t think the default value of `[]` makes sense for any of the `assertClassification` functions. If you use them, you almost certainly want to check that some classifications are produced.

---

What you think of naming this parameter `expected`. I think that would clean up the calls a little.

Reply:
The reason why I have an empty array as default value for classification spec is there are some test cases that gives a certain `ByteSourceRange` and there are only trivia in that range. And our `assertClassification` function filter out trivia.